BN9: QoL improvements.

Zelow79 opened the pull request:
`Bn9` now starts with the same node that `bn9.3` rewards (the idea here is, it helps showcase the BNs new feature and just speeds up a lot of what made it so slow early on).

Zelow79 committed:
bn9 QoL improvements

Increased BN multipliers for HacknetNodeMoney and WorldDaemonDifficulty to PI (3.14159...). This brings WD required hack to 9424.77796076938 (now the 2nd highest ^-^).
Bn9 now starts with the same node bn9.3 rewards (the idea here is it helps showcase the BNs new feature and just speeds up a lot of what made it so slow early on)

Zelow79 committed:
Update Prestige.ts

updated comment to be a little more clear

Mughur commented:
I think adding the free node is a good idea, but I'm not so sure about increasing the hash production and WD difficulty.
Also using pi just feels out of place :/

Zelow79 commented:
> I think adding the free node is a good idea, but I'm not so sure about increasing the hash production and WD difficulty. Also using pi just feels out of place :/

Updated to return BN values to original values.

Zelow79 committed:
Update BitNode.tsx

Removed PI from BN modifiers and set them back to base/original values as per Mughur request.

Zelow79 committed:
Update BitNode.tsx
